monitoring/configuration/ks-ctl-manager-svc.yaml

```diff
@@ -0,0 +1,12 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: ks-controller-manager
+  labels:
+    app: ks-controller-manager
+spec:
+  selector:
+    control-plane: controller-manager
+  ports:
+  - name: https
```

Comment:
Wouldn't "metrics" be a better name for this port?

Reply:
Fixed!